tests/trestle/core/models/file_content_type_test.py

```diff
@@ -40,23 +40,27 @@ def test_to_file_extension():
         FileContentType.to_file_extension(-1)
 
 
-def test_path_to_file_content_type(tmp_trestle_dir: Path):
+def test_path_to_file_content_type(tmp_trestle_dir: Path) -> None:
     """Test path_to_file_content_type method."""
     tmp_stem = tmp_trestle_dir / 'content_type_test'
 
     tmp_file = tmp_stem.with_suffix('.json')
     tmp_file.touch()
-    assert FileContentType.JSON == FileContentType.path_to_content_type(tmp_stem)
+    assert FileContentType.JSON == FileContentType.path_to_content_type(tmp_file)
+    assert FileContentType.path_to_file_extension(tmp_file) == '.json'
     tmp_file.unlink()
 
     tmp_file = tmp_stem.with_suffix('.yaml')
     tmp_file.touch()
-    assert FileContentType.YAML == FileContentType.path_to_content_type(tmp_stem)
+    assert FileContentType.YAML == FileContentType.path_to_content_type(tmp_file)
+    assert FileContentType.path_to_file_extension(tmp_file) == '.yaml'
     tmp_file.unlink()
 
     tmp_file = tmp_stem.with_suffix('.yml')
     tmp_file.touch()
-    assert FileContentType.YAML == FileContentType.path_to_content_type(tmp_stem)
+    assert FileContentType.YAML == FileContentType.path_to_content_type(tmp_file)
+    assert FileContentType.path_to_file_extension(tmp_file) == '.yml'
     tmp_file.unlink()
 
-    assert FileContentType.UNKNOWN == FileContentType.path_to_content_type(tmp_stem)
+    assert FileContentType.UNKNOWN == FileContentType.path_to_content_type(tmp_file)
+    assert FileContentType.path_to_file_extension(tmp_file) == ''
```

Comment:
This unknown type must have impacts in different parts of the code. Lets hope it works :)